### PR TITLE
C396396 Verify the inability to save empty statistical code field on Holdings create/edit page (folijet)

### PR DESCRIPTION
**Pull request by manowarqqq**
https://foliotest.testrail.io/index.php?/cases/view/396396
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4616/allure/

### CODE REVIEW

File: cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  before('Create test data', () => {
+    cy.createTempUser([Permissions.inventoryAll.gui]).then((userProperties) => {
+      testData.user = userProperties;
+      cy.getAdminToken()
+        .then(() => {
+          cy.getInstanceTypes({ limit: 1 });
+          cy.getInstanceIdentifierTypes({ limit: 1 });
+        })
+        .then(() => {
+          cy.createInstance({
```
Comment (ernestyedigaryan1) [inline cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js:35]: better to use the new approach for creating instances
example cypress/e2e/loans/multiple-loans.cy.js

**Review by dnovikau — APPROVED** [review]

File: cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      limit: 1,
+      query: `"instanceId"="${testData.item.instanceId}"`,
+    })
+      .then((holdings) => {
+        testData.item.holdingUUID = holdings[0].id;
+      })
+      .then(() => {
+        cy.deleteHoldingRecordViaApi(testData.item.holdingUUID);
+      });
+    InventoryInstance.deleteInstanceViaApi(testData.item.instanceId);
```
Comment (ernestyedigaryan1) [inline cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js:63]: And here also please use:
`InventoryInstances.deleteInstanceViaApi({ instance: folioInstances[0], servicePoint: testData.servicePoint, shouldCheckIn: true, });`

It also deletes holding record inside of it

File: cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    instanceName: `Inventory Instance ${randomFourDigitNumber()}`,
+    itemBarcode: randomFourDigitNumber(),
+  },
+  statisticalCode: 'Book, print (books)',
+  secondStatisticalCode: 'Books, electronic (ebooks)',
+  errorMessage: 'Please select to continue',
+  calloutMessage: 'has been successfully saved.',
+};
+
+describe('Holdings', () => {
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js:25]: please use 2 describes like: describe('inventory', () => {
  describe('Holdings', () => {

File: cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      });
+    InventoryInstance.deleteInstanceViaApi(testData.item.instanceId);
+    Users.deleteViaApi(testData.user.userId);
+  });
+
+  it(
+    'C396396 Verify the inability to save empty statistical code field on Holdings create/edit page (folijet) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+    () => {
+      // #1 Go to "Inventory" application -> Search and select the Instance from precondition
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js:71]: I think describing steps is unnecessary in tests

File: cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js (hunk 3 above)
Comment (manowarqqq) [reply cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js:25]: done

File: cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+      InventoryInstance.searchByTitle(testData.item.instanceName);
+      InventorySearchAndFilter.verifyInstanceDisplayed(testData.item.instanceName);
+      InventorySearchAndFilter.checkRowsCount(1);
+      InventoryInstances.selectInstance();
+      InventoryInstance.waitLoading();
+
+      // #2 Click "Add holdings" button
```
Comment (TetianaParanich) [inline cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js:78]: delete such comments please

**Review by TetianaParanich — APPROVED** [review]

File: cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js (hunk 2 above)
Comment (manowarqqq) [reply cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js]: This method failed when Inctance doesn't have an item

**Review by ernestyedigaryan1 — APPROVED** [review]

**Pull request merged**